### PR TITLE
Show Oppfolgingsplaner from LPS

### PR DESCRIPTION
**Pull request by janzh**
Show Oppfolgingsplaner from LPS that has a PersonOppgave with a refence to it. A personOppgave is created with a reference to a Oppfolgingsplan from LPS if Arbeidsgiver marked BehovForBistandFraNAV.

- Oppfolgingsplaner from LPS with with a unprocessed PersonOppgave are listed under "Aktive Oppfolgingsplaner".
- Oppfolgingsplaner from LPS with with a processed PersonOppgave are listed under "Tidligere Oppfolgingsplaner".
- Oppfolgingsplaner from LPS and their PDF's are request from syfooppfolgingsplanservice.
- PersonOppgaver are requested from Ispersonoppgave.

### CODE REVIEW

File: mock/mockSyfooppfolgingsplanservice.js [hunk 1]
```diff
@@ -20,6 +25,25 @@ function mockForLokal(server) {
...
+
+    server.get('/syfooppfolgingsplanservice/api/internad/dokument/lps/:uuid', (req, res) => {
+        const file = path.join(__dirname, '/oppfolgingsplan/pdf/oppfolgingsplanlps.pdf');
+        res.download(file, function (err) {
+            if (err) {
+                res.status(500).send('Error');
+            }
+        });
+
+    });
```
Comment (JMLindseth) [inline mock/mockSyfooppfolgingsplanservice.js:46]: Kult med pdf lokalt også 🎉

**Review by juneih — DISMISSED** [review]

File: mock/mockSyfooppfolgingsplanservice.js (hunk 1 above)
Comment (janzh) [reply mock/mockSyfooppfolgingsplanservice.js:46]: Ja, det er bra 🎉

File: src/components/oppfolgingsplan/lps/OppfolgingsplanLPSContent.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+};
+
+const OppfolgingsplanLPSContent = (
+    {
+        oppfolgingsplanLPS,
+    }
+) => {
+    return (
+        <div className="panel">
+            <p>Virksomhetsnavn: {oppfolgingsplanLPS.virksomhetsnummer}</p>
```
Comment (JMLindseth) [inline src/components/oppfolgingsplan/lps/OppfolgingsplanLPSContent.js:33]: Virksomhetsnavn -> Virksomhetsnummer ?

Comment (janzh) [reply src/components/oppfolgingsplan/lps/OppfolgingsplanLPSContent.js]: Fikset.

File: src/reducers/oppfolgingsplanerlps.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import { Reducer } from 'redux';
+import {
+    HENT_OPPFOLGINGSPLANER_LPS_FEILET,
+    HENT_OPPFOLGINGSPLANER_LPS_HENTER,
+    HENT_OPPFOLGINGSPLANER_LPS_HENTET
+} from '../actions/oppfolgingsplanerlps_actions';
+import { VIRKSOMHET_HENTET } from "../actions/virksomhet_actions";
+import { OppfolgingsplanLPS } from '../types/OppfolgingsplanLPS';
+import {PersonOppgave} from "../types/PersonOppgave";
```
Comment (JMLindseth) [inline src/reducers/oppfolgingsplanerlps.ts:9]: Mellomrom?

File: src/sagas/personoppgaveSagas.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        const personOppgaveList = data || []
+        yield put(actions.hentPersonOppgaverHentet(personOppgaveList));
+    } catch (e) {
+        yield put(actions.hentPersonOppgaverFeilet());
+    }
+}
+
+export const skalHentePersonOppgaver = (state: any) => {
+    const reducer = state.personoppgaver;
+    return !(reducer.henter || reducer.hentet || reducer.hentingFeilet);
```
Comment (JMLindseth) [inline src/sagas/personoppgaveSagas.ts:26]: Kan du ikke bruke `reducer.hentingForsokt` her?

File: src/reducers/oppfolgingsplanerlps.ts (hunk 3 above)
Comment (janzh) [reply src/reducers/oppfolgingsplanerlps.ts]: Fikset 👍

File: src/sagas/personoppgaveSagas.ts (hunk 4 above)
Comment (janzh) [reply src/sagas/personoppgaveSagas.ts]: Fikset her og i oppfolgingsplanerlpsSagas 👍

**Review by JMLindseth — APPROVED** [review]

**Pull request merged**